test/server/routes/auth.test.js

```diff
@@ -108,99 +110,31 @@ describe('AUTH', () => {
           done();
         });
     });
-  });
 
```

Comment:
What happened to all other tests in this file?

Reply:
they are there